.circleci/config.yml

```diff
@@ -337,6 +346,60 @@ jobs:
             - "dist/*linux_amd64.deb"
 
 
+  test-e2e-webui:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: large
+    steps:
+      - checkout
+      - run: sudo apt-get update && sudo apt-get install -y docker-ce docker-ce-cli containerd.io
+      - run: sudo systemctl restart docker
+      - build-docker
+
+      - run: pyenv global 3.6.5
+      - run: |
+          . /opt/circleci/.nvm/nvm.sh
+          nvm install v12
+          nvm alias default v12
+          make -C webui/tests get-deps
+      - run: pip install wheel
```

Comment:
we should do the pip stuff like https://github.com/determined-ai/determined/blob/master/.circleci/config.yml#L490

```
      - setup-python-venv:
          determined-common: true
          determined-cli: true
          executor: ubuntu-1604:201903-01
```
I think is the right one here